NoteFind/Delegates/AppDelegate.h

```diff
@@ -6,11 +6,12 @@
 //
 
 #import <UIKit/UIKit.h>
-
+#import <CoreData/CoreData.h>
```

Comment:
You can move this import to AppDelegate.m and just forward declare the NSPersistentContainer here

```
@class NSPersistentContainer;
```

Reply:
done!